kotlintest-tests/kotlintest-tests-core/src/test/kotlin/com/sksamuel/kotlintest/EventuallyTest.kt

```diff
@@ -71,6 +72,13 @@ class EventuallyTest : WordSpec() {
         }
         result shouldBe 1
       }
+      "display the underlying assertion failure" {
+        shouldThrow<AssertionError> {
+          eventually(Duration.ofSeconds(2), AssertionError::class.java) {
```

Comment:
I don't think this Duration must be that great. The assertion is so simple, it can be reduced to `Duration.ofMillis(10)` and it will still work correctly

Reply:
Having a big duration on this test will add more time to the test suite that we don't really need. If we try to reduce this time everywhere, we may have a much faster test suite!

Reply:
I have changed the timeout and will now merge.